src/domain-registry-essentials/templates/cns-smart-contracts-renderer.ts

```diff
@@ -0,0 +1,98 @@
+import Fs from 'fs';
+import Ejs from 'ejs';
+import MarkdownInclude from 'markdown-include';
+import NetworkConfigJson from 'dot-crypto/src/network-config/network-config.json';
+
+const TemplatesDir = 'src/domain-registry-essentials/templates';
+const ContractsDir = TemplatesDir + '/contracts';
+const MarkdownFile = TemplatesDir + '/markdown/cns-smart-contracts-markdown.json';
```

Comment:
I'd use https://nodejs.org/api/path.html#path_path_join_paths to stop caring about putting delimiters correctly.

Reply:
Thanks, replaced this.